calculator/calculator.py

```diff
@@ -0,0 +1,20 @@
+print("Select Operation:\n1- Add,\n2- Subtract,\n3- Multiply,\n4- Divide.");
+option = int(input("Enter you choice from 1/2/3/4 :- "));
+num1 = input("Enter your first number :- ");
+num2 = input("Enter your second number :- ");
+if option == 1:
+    {
```

Comment:
This is not required in python to define the scope of the nested code. Python defines scope using indentation.
```
if option == 1:
    print("results : ", int(num2)+int(num1))
```

Reply:
okay